components/gitpod-protocol/src/permission.ts

```diff
@@ -53,15 +48,9 @@ export namespace RolesOrPermissions {
 }
 
 export namespace Permission {
-    /** The permission to monitor the (live) state of a Gitpod installation */
-    export const MONITOR: PermissionName = "monitor";
-
     /** The permission to develop on this running Gitpod installation */
     export const DEVELOPER: PermissionName = "developer";
 
-    /** The permission for actions like block user, stop workspace, etc. */
-    export const ENFORCEMENT: PermissionName = "enforcement";
-
     /** The permission for registry access (start workspaces referencing gitpod-internal Docker images) */
     export const REGISTRY_ACCESS: PermissionName = "registry-access";
```

Comment:
Is this still used/required? @WVerlaek Maybe you can tell? Only usage seems to be [here](https://github.com/gitpod-io/gitpod/blob/097d178910f860d932d3c5343a334db156e7d948/components/server/src/workspace/workspace-starter.ts#L1003). :thinking:

Reply:
Not sure, but seems to be used for the DevOps role [here](https://github.com/gitpod-io/gitpod/blob/1a663acfd31c1c8c7df0a25cebc1abfd66cbcce7/components/gitpod-protocol/src/permission.ts#L102)? I don't know if that's in use though

Reply:
I checked the DB, and the occurrences there seemed to be leftovers only.

I was more interested of the impact/usage of the [BuildRegistryAuthTotal](https://github.com/gitpod-io/gitpod/blob/097d178910f860d932d3c5343a334db156e7d948/components/server/src/workspace/workspace-starter.ts#L1005-L1006C62)v construct: Is that implemented in image-builder-mk3 at all? Is it used anywhere concretely?

Reply:
> do you know if that role is also no longer in use

It still has some entries in the prod DB, but they look like leftovers. But it's a role, only an aggregation of permissions, so we can replace it with "registry-access" if we really need to.
We might have to ask Chris on why we have it in the first place... :+1: